cg/cli/clean.py

```diff
@@ -449,10 +450,10 @@ def remove_old_flow_cell_run_dirs(context: CGConfig, sequencer: str, days_old: i
 def remove_old_demutliplexed_run_dirs(context: CGConfig, days_old: int, dry_run: bool):
     """Removes flow cells from demultiplexed run directory."""
     status_db: Store = context.status_db
-    demux_api: DemultiplexingAPI = context.demultiplex_api
+    demux_post_processing_api: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
```

Comment:
I think we should stick to the pattern of setting up the `apis` in the context and not instantiate them directly where they are used. Probably out of scope for this PR

Reply:
I opened an issue regardin this